### ISSUE TITLE
Adding Boosters or Effects introduced by other Mods.

### ISSUE THREAD

**Issue by xdrcix** [NovFR/DoS-2-Savegame-Editor#20]
So I've noticed that the list of Boosters is pulled from the Datas\Boosters.txt file, however it seems that this file doesn't extract boosters that are applied from mods.

I was curious as to how it would be possible to add in these boosters manually. Would it involve me knowing the exact name of the booster then editing the txt file?

**Comment by NovFR** [comment]
Yes.

The easiest way is to have an object that contains the booster and edit it. It will appear in the list with its actual name. Copy this name and add it directly into the text file, the booster will then be available when you click Add.

You can also edit the databases to change the names, but this is not required.

**Comment by xdrcix** [comment]
Cool ! I had a feeling that was the best way.

I do have another question though. I'm using the Runy Tunes mod, which by its name adds in a few new runes into the game and I would like to also add in those new runes into the save editor. So similar to the boosters, I placed the names into the rune.txt file in the Datas directory, but I noticed that the actual icons don't show up. So how exactly can I go about editing the runes.icl file?
